docs/report.md

```diff
@@ -72,8 +72,13 @@ State which software license you chose for your application.
 
 ### LLMs, ChatGPT, CoPilot, and others
 
-State which LLM(s) were used during development of your project.
-In case you were not using any, just state so.
-In case you were using an LLM to support your development, briefly describe when and how it was applied.
-Reflect in writing to which degree the responses of the LLM were helpful.
-Discuss briefly if application of LLMs sped up your development or if the contrary was the case.
+In the development process LLM were used sparingly to support the coding process.
+GitHub copilot and the JetBrains AI was occasionally used to finish lines of code, when it came with good suggestions. 
+It also wrote some of the documentation for the code and helped with some tests.
+ChatGPT was used occasionally to suggest names, explain error messages, and other similar uses.
+
+Often the answers were wrong or irrelevant though, especially regarding ChatGPT, however it rarely took long to figure out whether the answer was useful, making it a small problem.
+It was helpful as support and probably sped up the coding process, but the final product most likely did not change because of it.
+Since neither ever contributed significantly to the codebase it has not been added as a co-author to any commits.
```

Comment:
Would it be possible to rewrite this sentence, it sounds weird saying neither ever contributed significantly, since we and censor might have different opinions on what significantly means.
Just to make it more clear, on why we have decided on not to include it

Reply:
I can definitly rewrite this if you want, but if we bring in our thoughts I either need to say "we" or refer to us as "the developers" or "the group".